src/dynamodb.js

```diff
@@ -66,14 +66,13 @@ async function destroyTables (dynamoClient, uniqueIdentifier) {
   }
 }
 
-async function getConnection () {
-  if (process.env.DYNAMODB_ENDPOINT) {
-    return buildConnectionAndConfig({ url: process.env.DYNAMODB_ENDPOINT });
-  }
-
-  const docker = new Docker();
-  const environment = new Environment();
-
+/**
+ * @param {object} opts
+ * @param {object} opts.docker Used to mock the Docker library in tests
+ * @param {boolean} opts.inMemory Whether to run local DynamoDB in in-memory mode.
+ * 'false' persists data to disk.
+ */
+async function launchDynamoContainer ({ docker = new Docker(), inMemory = false } = { docker: new Docker(), inMemory: false }) {
```

Comment:
Question, would `{ docker = new Docker(), inMemory = false } = { }` achieve the same thing without duplication?